projects/WTI-API/src/main/controllers/MainController.java

```diff
@@ -14,6 +14,7 @@
 import edu.csus.ecs.pc2.api.IProblem;
 import edu.csus.ecs.pc2.api.ServerConnection;
 import edu.csus.ecs.pc2.api.implementation.Contest;
+import edu.csus.ecs.pc2.api.implementation.ProblemImplementation;
```

Comment:
This should be removed as it is not used (Eclipse flags it as a warning).

Reply:
I know how this happened.  Originally the lengthy comments I had added included a javadoc reference like `{@link ProblemImplementation}`.  This caused Eclipse to add the import (since it would be needed by javadoc).  Later, in an effort to reduce the complexity of the comments, I removed some of those "{@link...}" references. However, I didn't notice that the import still remained.  I just pushed a commit that removes it.